cli/src/pcluster/config/update_policy.py

```diff
@@ -481,6 +545,10 @@ def condition_checker_login_nodes_stop_policy(_, patch):
     "managed_placement_group": actions_needed_managed_placement_group,
     "shared_storage_update_conditional": actions_needed_shared_storage_update,
     "managed_fsx": actions_needed_managed_fsx,
+    "compute_nodes_stop": lambda change, patch: (
```

Comment:
why this "compute_nodes_stop" is under "condition_checker_LOGIN_NODES_pools_policy"?

Reply:
Good catch, addressing